### DISCUSSION TITLE
N tane Vezir

### DISCUSSION THREAD

**Discussion by OmerAkbayrak** [sonsuzus/sonsuzus#385] [Soru Cevap]
**Problem**
N x N kadar kareden oluşan bir satranç tahtası vereceksiniz ve N tane vezir yerleştireceksiniz. Ve vezirlerin diğer herhangi bir vezire saldırı yapamayacak pozisyonda olması gerekecek.

**Girdi:**
Tek bir satırda N değişkeni için tek bir sayı değeri
**Çıktı:**
Eğer mümkünse tüm vezirleri diğerlerine saldırı yapamayacak şekilde yerleştireceksiniz, sonra N tane sayı içeren N tane satır yazdıracaksınız. i. satırdaki ve j. sütundaki sayı eğer vezir oraya yerleştirilecekse 1'i gösterecek aksi takdirde 0 olacak. Eğer vezirleri yerleştirmenin başka bir yolu varsa herhangi birini yazdırın, eğer tüm vezirleri doğru şekilde yerleştirmek mümkün değilse "Mümkün Değil" yazdırın.
**Sınırlamalar:**
1<N<10.

**_Sorunun_ Linki: _https://www.hackerearth.com/practice/basic-programming/recursion/recursion-and-backtracking/practice-problems/algorithm/n-queensrecursion-tutorial/_**

**Comment by sonsuzus** [comment]
Sample Input
4
Sample Output
```
0 1 0 0 
0 0 0 1 
1 0 0 0 
0 0 1 0 
```
